stories/components/SearchBar.stories.js

```diff
@@ -0,0 +1,10 @@
+import SearchBar from "../../components/SearchBar";
+
+export default {
+  component: SearchBar,
+  title: "components/Search Bar",
```

Comment:
What is `title` for? Other stories don't seen to have this?

Reply:
If I don't use `title`, storybook uses the story file name and it is without a space between "Search" and "Input".

![image](https://user-images.githubusercontent.com/27003616/222450071-95297511-bc37-4f49-b99c-7959668f0d3c.png)